mix.exs

```diff
@@ -19,16 +19,15 @@ defmodule Ueberauth.Mixfile do
   end
 
   def application do
-    [applications: [:logger]]
+    [extra_applications: [:logger]]
   end
 
   defp deps do
-    [{:plug, "~> 1.2"},
+    [{:plug, "~> 1.5.0"},
 
-     # dev/test dependencies
-     {:credo, "~> 0.8.10", only: [:dev, :test]},
-     {:earmark, "~> 0.2", only: :dev},
-     {:ex_doc, "~> 0.12", only: :dev}]
+      # dev/test dependencies
+      {:credo, "~> 0.9.0", only: [:dev, :test]},
```

Comment:
Let's drop credo since we have the formatter now 👍

Reply:
credo is dropped. Thanks for the review. :+1: